sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -1319,3 +1332,40 @@ async def read_feed_ranges(
 
         return [FeedRangeEpk(Range.PartitionKeyRangeToRange(partitionKeyRange))
                 for partitionKeyRange in partition_key_ranges]
+
+    async def get_updated_session_token(self,
+                                  feed_ranges_to_session_tokens: List[Tuple[str, FeedRange]],
+                                  target_feed_range: FeedRange
+                                  ) -> "Session Token":
```

Comment:
Looks like the formatting is a bit wonky here.

Reply:
Also, `"Session Token"` isn't a valid type hint.

Reply:
Fixed formatting and the type hint